pmda/test/test_leaflet.py

```diff
@@ -39,24 +38,29 @@ def correct_values(self):
     def correct_values_single_frame(self):
         return [np.arange(1, 2150, 12), np.arange(2521, 4670, 12)]
 
-    def test_leaflet(self, universe, correct_values):
+    # XFAIL for 2 jobs needs to be fixed!
+    @pytest.mark.parametrize('n_jobs', (1, pytest.mark.xfail(2)))
```

Comment:
@iparask the `test_leaflets` test failed for me with n_jobs=2
```
E   AssertionError:
E   Arrays are not almost equal to 7 decimals
E   error: leaflets should match test values
E   (shapes (1,), (6,) mismatch)
E    x: array([36634])
E    y: array([36507, 36761, 37523, 37650, 38031, 38285])
```

My expectation was that this should give the same answer, just run faster... Can you please look into this?

Reply:
See also #76